andela/ah-frontend-realers
# Chore(Test Environment): Set up test environment

#### What does this PR do?

Set up testing environment for project

#### Description of Task to be completed?

Install jest and enzyme
Create setupTest.js file
Set up testing environment

#### How should this be manually tested?

* -	Git fetch this repo locally in the terminal with `git fetch origin ch-setup-test-environment-166187810
* -	Install application dependencies: `npm install`
* -	Run application tests: `npm test`
* -	Check test coverage: `npx jest -–coverage`

#### Any background context you want to provide?

This application did not have a testing environment nor tests, I have set up both.

#### What are the relevant pivotal tracker stories?

#[166187810](https://www.pivotaltracker.com/story/show/166187810)

#### Screenshots

Passing Tests with the coverage

<img width="856" alt="Screenshot 2019-07-02 at 18 11 19" src="https://user-images.githubusercontent.com/32802973/60525924-0028b100-9cf8-11e9-88d0-cfe45bbc611c.png">
